### PR TITLE
NO-ISSUE - Log output from bip installer gather

### PR DESCRIPTION
**Pull request by omertuc**

### CODE REVIEW

**Review by osherdp — APPROVED** [review]
/lgtm

File: discovery-infra/bootstrap_in_place.py [hunk 1]
```diff
@@ -38,10 +40,16 @@ def installer_generate(openshift_release_image):
 
 @utils.retry(exceptions=Exception, tries=5, delay=30)
 def installer_gather(ip, ssh_key, out_dir):
-    _stdout, stderr, _ret = utils.run_command(
-        f"{INSTALLER_BINARY} gather bootstrap --bootstrap {ip} --master {ip} --key {ssh_key}"
+    stdout, stderr, _ret = utils.run_command(
```
Comment (eranco74) [inline discovery-infra/bootstrap_in_place.py:43]: Consider refactoring utils.run_command to get optional stdout and stderr.

Comment (omertuc) [reply discovery-infra/bootstrap_in_place.py:43]: Not sure what you mean. It's already returning stdout and stderr?

**Pull request merged**